### ISSUE TITLE
Support deeper levels of filter links

### ISSUE THREAD

**Issue by aleksejrs** [openfoodfacts/openfoodfacts-server#546]
http://world.openfoodfacts.org/brand/purina/country/france filters by *brand* and *country*, but http://world.openfoodfacts.org/brand/purina/country/france/category/bla does not filter by *category*.

**Comment by TaciteOFF** [comment]
In the meantime, you can display the results using the search page.

**Comment by hangy** [comment]
Yes, I've seen this limitation, too. The two levels are actually hard-coded in `Display.pm`, as far as I can see.

**Comment by teolemon** [comment]
SEO issues me thinks. We could enable it for the API though.

**Comment by hangy** [comment]
AFAIK, the 2nd level is excluded from indexing, so that should not really matter.
